apps/web/src/components/Profile/NftGallery/Create.tsx

```diff
@@ -0,0 +1,60 @@
+import EmojiPicker from '@components/Shared/EmojiPicker';
+import { Button } from '@components/UI/Button';
+import { Modal } from '@components/UI/Modal';
+import { ChevronLeftIcon } from '@heroicons/react/outline';
+import { Trans } from '@lingui/macro';
+import clsx from 'clsx';
+import type { Dispatch, FC } from 'react';
+import React, { useState } from 'react';
+
+import Picker from './Picker';
+
+type Props = {
+  showModal: boolean;
+  setShowModal: Dispatch<boolean>;
+};
+
+const Create: FC<Props> = ({ showModal, setShowModal }) => {
+  const [emoji, setEmoji] = useState('');
+  const [pickNfts, setPickNfts] = useState(false);
+
+  return (
+    <div>
```

Comment:
do we need div here? or just fragment `<>`?

Reply:
not requried, removing